src/assets/constants.js

```diff
@@ -96,74 +104,26 @@ export const websiteLogoMapGrey = {
   footlocker: FootlockerGrey
 }
 
-// browse - single-select filter options
-
-export const popularBrandOptions = [
-  { value: 'nike', label: 'Nike' },
-  { value: 'yeezy', label: 'Yeezy' },
-  { value: 'air jordan', label: 'Air Jordan' },
-  { value: 'off white', label: 'Off-White' },
-  { value: 'adidas', label: 'Adidas' },
-  { value: 'converse', label: 'Converse' }
+export const sneakerSizes = [
+  4, 4.5, 5, 5.5, 6, 6.5, 7, 7.5, 8, 8.5, 9,
+  9.5, 10, 10.5, 11, 11.5, 12, 12.5, 13, 14, 15
 ]
 
-export const brandOptions = [
-  { value: null, label: 'All' },
-  { value: 'adidas', label: 'Adidas' },
-  { value: 'asics', label: 'Asics' },
-  { value: 'balenciaga', label: 'Balenciaga' },
-  { value: 'converse', label: 'Converse' },
-  { value: 'fila', label: 'Fila' },
-  { value: 'gucci', label: 'Gucci' },
-  { value: 'air jordan', label: 'Jordan' },
-  { value: 'new balance', label: 'New Balance' },
-  { value: 'nike', label: 'Nike' },
-  { value: 'off white', label: 'Off-White' },
-  { value: 'puma', label: 'Puma' },
-  { value: 'reebok', label: 'Reebok' },
-  { value: 'saucony', label: 'Saucony' },
-  { value: 'supreme', label: 'Supreme' },
-  { value: 'under armour', label: 'Under Armour' },
-  { value: 'vans', label: 'Vans' },
-  { value: 'yeezy', label: 'Yeezy' }
+export const brands = [
+  'adidas', 'Air Jordan', 'ASICS', 'Converse',
+  'New Balance', 'Nike', 'Puma', 'Reebok', 'Under Armour'
 ]
 
-export const sizeTypeOptions = [
-  { value: null, label: 'All' },
-  { value: 'men', label: 'Men' },
-  { value: 'women', label: 'Women' },
-  { value: 'youth', label: 'Youth' },
-  { value: 'infant', label: 'Infant' }
+export const sortByOptions = [
+  'relevance', 'trending', 'newest', 'price_low', 'price_high'
 ]
 
-// sort options
-export const sortOptions = [
-  { value: null, label: 'All' },
-  { value: '{"sort": "date_added", "order": "descending"}', label: 'Newest Releases' },
-  { value: '{"sort": "relevance", "order": "descending"}', label: 'Most Popular' },
-  { value: '{"sort": "lowest_price_cents", "order": "ascending"}', label: 'Price: Low to High' },
-  { value: '{"sort": "lowest_price_cents", "order": "descending"}', label: 'Price: High to Low' }
-]
-
-// browse - multi-select filter options
-function generateReleaseYears () {
-  const currentYear = new Date().getFullYear()
-  const years = []
-  for (let i = currentYear; i >= 2000; i--) {
-    years.push({ value: i.toString(), label: i.toString() })
-  }
-  return years
+export const sortByOptionsFormattedMap = {
```

Comment:
Change name to `sortByOptionToDisplayText`? It's same # of chars but I think more descriptive